bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,85 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+import json
+
+
+@linter(executable='csscomb',
+        output_format='corrected',
+        use_stdout=True,
+        use_stdin=False,
+        result_message='Change coding style based on CSS rules written in ' +
+                       '.csscomb.json',
+        )
+class CSSCombBear:
+    """
+    CSScomb is a coding style formatter for CSS.
+
+    For more information, consult <https://github.com/csscomb/csscomb.js>.
+    """
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/bxpke7nizyxdf6mlx6ss4h405'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file, csscomb_json: str = ""):
+        args = (filename,)
+        if csscomb_json:
+            args += (csscomb_json,)
+        else:
+            args += (config_file,)
+        return args
+
+    def generate_config(filename, file,
+                        verbose: bool = True,
+                        remove_empty_rulesets: bool = True,
+                        always_semicolon: bool = True,
+                        color_case: str = "upper",
+                        block_indent: str = "\\t",
+                        color_shorthand: bool = False,
+                        eof_newline: bool = True,
+                        leading_zero: bool = False,
+                        quotes: str = "single",
+                        sort_order_fallback: str = "abc",
+                        space_before_colon: str = " ",
+                        space_after_colon: str = " ",
+                        space_between_declar: str = "\\n",
+                        space_before_o_brace: str = "\\n",
+                        space_after_o_brace: str = "\\n",
+                        space_after_s_delim: str = "\\n",
+                        space_before_s_delim: str = " ",
+                        space_before_c_brace: str = "\\n",
+                        strip_spaces: bool = True,
+                        tab_size: bool = True,
+                        vendor_prefix_align: bool = True,
+                        csscomb_json: str = ""):
+
+        if not csscomb_json:
+            options = {"verbose": verbose,
```

Comment:
hey @Asnelchristian , look, options! 

im sure you can give him better guidance for namings and documenting than anyone else :D